Adding ARM Disk Management cmdlets

WillEastbury opened the pull request:
Previously the ASM cmdlets were in the doc, added the matching ARM cmdlets to set VM disk options.

azurecla commented:
Hi __@harelbr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

WillEastbury committed:
Adding ARM Disk Management cmdlets 

Previously the ASM cmdlets were in the doc, added the matching ARM cmdlets to set VM disk options.

rothja commented on articles/virtual-machines/windows/sql/virtual-machines-windows-sql-performance.md:
```diff
@@ -110,7 +110,7 @@ For VMs that support Premium Storage (DS-series, DSv2-series, and GS-series), we
   * If you are not using Premium Storage (dev/test scenarios), the recommendation is to add the maximum number of data disks supported by your [VM size](../sizes.md?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json) and use Disk Striping.
 
-* **Caching policy**: For Premium Storage data disks, enable read caching on the data disks hosting your data files and TempDB only. If you are not using Premium Storage, do not enable any caching on any data disks. For instructions on configuring disk caching, see the following topics: [Set-AzureOSDisk](https://msdn.microsoft.com/library/azure/jj152847) and [Set-AzureDataDisk](https://msdn.micro ...
+* **Caching policy**: For Premium Storage data disks, enable read caching on the data disks hosting your data files and TempDB only. If you are not using Premium Storage, do not enable any caching on any data disks. For instructions on configuring disk caching, see the following topics: (For the classic (ASM) deployment model see: [Set-AzureOSDisk](https://msdn.microsoft.com/library/azure/jj15284 ...
```
This change is good, but please change the formatting to:

* **Caching policy**: For Premium Storage data disks, enable read caching on the data disks hosting your data files and TempDB only. If you are not using Premium Storage, do not enable any caching on any data disks. For instructions on configuring disk caching, see the following topics. For the classic (ASM) deployment model see: [Set-AzureOSDisk](https://msdn.microsoft.com/library/azure/jj152847) and [Set-AzureDataDisk](https://msdn.microsoft.com/library/azure/jj152851.aspx). For the Resource Manager (ARM) deployment model see: [Set-AzureRMOSDisk](https://docs.microsoft.com/powershell/module/azurerm.compute/set-azurermvmosdisk?view=azurermps-4.4.1) and [Set-AzureRMVMDataDisk](https://docs.microsoft.com/powershell/module/azurerm.compute/set-azurermvmdatadisk?view=azurermps-4.4.1).

rothja replied:
Also, note that I took the en-us out of the URLs. It is hard to see because it does not show my RAW changes. I also removed an extra period at the end and an extra paren in the middle.

WillEastbury committed:
Requested formatting changes

Tweaks to sentence structure and punctuation as requested.

GitHubber17 committed:
Update virtual-machines-windows-sql-performance.md
